lib/eventFactory.js

```diff
@@ -108,20 +108,22 @@ function getJobsFromPR(config) {
 /**
  * Starts the build if the changed file is part of the sourcePaths, or if there is no sourcePaths
  * @method startBuild
- * @param  {Object}   config                configuration object
- * @param  {Object}   config.buildConfig    Build Config to create the build with
- * @param  {Array}    config.changedFiles   List of files that were changed
- * @param  {Array}    config.sourcePaths    List of soure paths
- * @param  {Boolean}  [config.webhooks]     If the create came from a webhook (pr or push) or not
- * @param  {Boolean}  [config.isPR]         Is it PR?
+ * @param  {Object}   config                    configuration object
+ * @param  {Object}   config.buildConfig        Build Config to create the build with
+ * @param  {Array}    config.changedFiles       List of files that were changed
+ * @param  {Array}    config.sourcePaths        List of soure paths
+ * @param  {Boolean}  [config.webhooks]         If the create came from a webhook (pr or push) or not
+ * @param  {Boolean}  [config.isPR]             Is it PR?
+ * @param  {Object}   [config.decoratedCommit]  Decorated commit object
  * @return {Promise}
  */
 function startBuild(config) {
     /* eslint-disable global-require */
     const BuildFactory = require('./buildFactory');
     const buildFactory = BuildFactory.getInstance();
     /* eslint-enable global-require */
-    const { buildConfig, changedFiles, sourcePaths, webhooks, isPR } = config;
+    const { buildConfig, changedFiles, sourcePaths, webhooks, isPR, decoratedCommit } = config;
+    const { author, committer, message } = decoratedCommit;
```

Comment:
do we want to get the `url` for meta too?

Reply:
guess there is no harm to can add that, will add it